waiter/src/waiter/process_request.clj

```diff
@@ -364,7 +369,7 @@
           (histograms/update! (metrics/service-histogram service-id "request-content-length") content-length)
           (statsd/inc! metric-group "request_content_length" content-length)))
       (catch Exception e
-        (log/error e "Unable to track content-length on request")))
```

Comment:
Just make all these capitalizations a separate PR. Search systematically. Then it will also cover the ```(log/error "there was a generic error``` from above

Reply:
We are switching to lower-case. This PR changes the instances in this file.

Reply:
oops

Reply:
Let's open a task to replace ```log/(error|info|warn) .* ?"[A-Z]```

Reply:
Created: https://github.com/twosigma/waiter/issues/825